docs/sso/overview.rst

```diff
@@ -0,0 +1,48 @@
+General Overview
+================
+
+What is SSO?
+------------
+
+Single sign-on (SSO) defines a scheme for authenticating a user with a single
+username and password across several related, but independent, applications.
+
+.. note::
+    Pay attention the difference between **authentication** and **authorization**.
+
+    **Authentication** is the process used to identity whether a user is who they say they are.
+
+    **Authorization** is the set of permissions given to an authenticated user on a system
```

Comment:
```suggestion
    **Authorization** is the set of permissions given to an authenticated user
```
Saying "on a system" is probably only needed once in this sentence, fine removing the other one instead if that sounds better.